data/kml_test_data/generated_mixed_tracks.kml

```diff
@@ -0,0 +1,326 @@
+      <mwm:tags>
+        <mwm:value>mountains</mwm:value>
+        <mwm:value>bike</mwm:value>
+      </mwm:tags>
+      <mwm:toponyms>
+        <mwm:value>10</mwm:value>
+        <mwm:value>11</mwm:value>
+      </mwm:toponyms>
+      <mwm:languageCodes>
+        <mwm:value>en</mwm:value>
+        <mwm:value>ja</mwm:value>
+        <mwm:value>ru</mwm:value>
+      </mwm:languageCodes>
+      <mwm:properties>
+        <mwm:value key="property1">value1</mwm:value>
+        <mwm:value key="property2">value2</mwm:value>
+      </mwm:properties>
+    </mwm:compilation>
+  </ExtendedData>
+  <Placemark>
+    <name>Мое любимое место</name>
+    <description>Test bookmark description</description>
+    <TimeStamp><when>1970-01-01T00:13:20Z</when></TimeStamp>
+    <styleUrl>#placemark-blue</styleUrl>
+    <Point><coordinates>45.9242,49.326859</coordinates></Point>
+    <ExtendedData xmlns:mwm="https://omaps.app">
+      <mwm:name>
+        <mwm:lang code="ru">Тестовая метка</mwm:lang>
+        <mwm:lang code="default">Test bookmark</mwm:lang>
+      </mwm:name>
+      <mwm:description>
+        <mwm:lang code="ru">Тестовое описание метки</mwm:lang>
+        <mwm:lang code="default">Test bookmark description</mwm:lang>
+      </mwm:description>
+      <mwm:featureTypes>
+        <mwm:value>historic-castle</mwm:value>
+        <mwm:value>historic-memorial</mwm:value>
+      </mwm:featureTypes>
+      <mwm:customName>
+        <mwm:lang code="en">My favorite place</mwm:lang>
+        <mwm:lang code="default">Мое любимое место</mwm:lang>
+      </mwm:customName>
+      <mwm:scale>15</mwm:scale>
+      <mwm:boundTracks>
+        <mwm:value>0</mwm:value>
+      </mwm:boundTracks>
+      <mwm:visibility>0</mwm:visibility>
+      <mwm:nearestToponym>12345</mwm:nearestToponym>
+      <mwm:minZoom>10</mwm:minZoom>
+      <mwm:properties>
+        <mwm:value key="bm_property1">value1</mwm:value>
+        <mwm:value key="bm_property2">value2</mwm:value>
+        <mwm:value key="score">5</mwm:value>
+      </mwm:properties>
+      <mwm:compilations>1,2,3,4,5</mwm:compilations>
+    </ExtendedData>
+  </Placemark>
+  <Placemark>
+    <name>Test track</name>
+    <description>Test track description</description>
+    <Style><LineStyle>
+      <color>FF0000FF</color>
+      <width>6</width>
+    </LineStyle></Style>
+    <TimeStamp><when>1970-01-01T00:15:00Z</when></TimeStamp>
+    <LineString><coordinates>45.9242,49.326859,1 45.2244,48.941288,2 45.1964,49.401948,3</coordinates></LineString>
+    <gx:MultiTrack>
+      <gx:Track>
+        <when>1970-01-01T00:00:00Z</when>
+        <when>1970-01-01T00:00:01Z</when>
+        <when>1970-01-01T00:00:02Z</when>
+        <gx:coord>45.9242 49.326859 1</gx:coord>
+        <gx:coord>45.2244 48.941288 2</gx:coord>
+        <gx:coord>45.1964 49.401948 3</gx:coord>
+      </gx:Track>
+      <gx:Track>
+        <when>1970-01-01T00:00:00Z</when>
+        <when>1970-01-01T00:00:01Z</when>
```

Comment:
It would be great to also test such cases:

when
gx:coord
when
gx:coord

and 

gx:coord
when
gx:coord
when

and

gx:coord
gx:coord
when
when

Reply:
And also add a case for the non-matching number of when & gx:coord

Reply:
> And also add a case for the non-matching number of when & gx:coord

See test case `Kml_Track_Points_And_Timestamps_Sizes_Mismatch`

Reply:
> It would be great to also test such cases:

Fixed!